client/my-sites/marketplace/components/reviews-modal/styles.scss

```diff
@@ -84,8 +84,3 @@ $border-color: #eee;
 		fill: var(--studio-yellow-20);
 	}
 }
-
-.dialog__action-buttons-close {
-	padding: 16px 24px 5px 5px;
-	background-color: #fff;
-}
```

Comment:
I removed these styles so that the plugin Dialog uses the default component styles.